controllers/RequestController.php

```diff
@@ -1,18 +1,12 @@
 <?php
-require_once 'ParkingDBController.php';
 require_once 'routes.php';
-require_once '../models/User.php';
-require_once '../models/UserService.php';
+require_once __DIR__ . '/../models/User.php';
+require_once __DIR__ . '/../models/UserService.php';
+require_once __DIR__ . '/../models/ParkingSpotService.php';
+require_once __DIR__ . '/../models/CourseService.php';

```

Comment:
Doesn't this mix absolute and relative paths into one?

For example, if the` __DIR__` is `/usr/lib` the resulting path will be:
`/usr/lib/../models/User.php`

Reply:
__DIR__ always returns the directory that 'User.php' is in. So there should not be a problem when project is deployed properly, because `__DIR__ . '/../models/` will always be correct.